app/src/main/java/com/lagradost/cloudstream3/ui/result/ResultFragmentTv.kt

```diff
@@ -327,6 +327,78 @@ class ResultFragmentTv : Fragment() {
                 }
             }
 
+            resultPlayMovieButton.setOnFocusChangeListener { _, hasFocus ->
```

Comment:
bro, wth am I seeing??
make this a variable and do smth like let focuschange = { v,f -> v.isSelected = f }

for v in listof(resultPlayMovieText,resultPlaySeriesButton...) { v.setOnFocusChangeListener(focuschange) }

Reply:
because rn it is wet code

Reply:
totally right, resolved